tilavarauspalvelu/settings.py

```diff
@@ -492,6 +492,8 @@ def SEARCH_SETTINGS(cls):
     REMOVE_RESERVATION_STATS_OLDER_THAN_YEARS = 5
     REMOVE_RECURRING_RESERVATIONS_OLDER_THAN_DAYS = 1
 
+    APPLICATION_ROUND_RESERVATION_CREATION_TIMEOUT_MINUTES = 10
```

Comment:
Should probably make this configurable via the environment, e.g. `values.IntegerValue(default=10)`.